### PR TITLE
Fix misleading description of syscalls checked on audit_rules_kernel_module_loading

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Update description of rule `audit_rules_kernel_module_loading`
- Add tests for various syscall arrangements within the audit rule

#### Rationale:

- Description of the rule is misleading for non-rhel6 systems. On rhel6 system, OVAL check for this rule requires audit rule for `init_module` and `delete_module` syscalls. The OVAL check for other systems will also require rules for `finit_module` and `create_module` syscalls.

- Related to #3210

### CODE REVIEW

**Comment by cipherboy** [comment]
(Travis CI build passed, idk why status isn't being updated).

**Comment by rshade** [comment]
@yuumasato I bet if you close this and open it the PR build will get marked good.

**Comment by yuumasato** [comment]
@openscap-ci test this please

**Comment by yuumasato** [comment]
Travis test passed: https://travis-ci.org/OpenSCAP/scap-security-guide/builds/418726576

File: tests/data/group_system/group_auditing/group_auditd_configure_rules/group_audit_kernel_module_loading/rule_audit_rules_kernel_module_loading/syscalls_multiple_per_arg.pass.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# Use auditctl, on RHEL7, default is to use augenrules
+sed -i "s%^ExecStartPost=.*%ExecStartPost=-/sbin/auditctl%" /usr/lib/systemd/system/auditd.service
+
+rm -f /etc/audit/rules.d/*
+
+# cut out irrelevant rules for this test
+sed '1,10d' test_audit.rules > /etc/audit/audit.rules
+sed -i '5,8d' /etc/audit/audit.rules
+cat /etc/audit/audit.rules
+true
```
Comment (mildass) [inline tests/data/group_system/group_auditing/group_auditd_configure_rules/group_audit_kernel_module_loading/rule_audit_rules_kernel_module_loading/syscalls_multiple_per_arg.pass.sh]: What's the purpose of this line and line above (`cat` call) in test?

Comment (yuumasato) [reply tests/data/group_system/group_auditing/group_auditd_configure_rules/group_audit_kernel_module_loading/rule_audit_rules_kernel_module_loading/syscalls_multiple_per_arg.pass.sh]: Well noted, they are leftovers of `sed` testing.
I have removed them.

File: tests/data/group_system/group_auditing/group_auditd_configure_rules/group_audit_kernel_module_loading/rule_audit_rules_kernel_module_loading/syscalls_multiple_per_arg.pass.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+# Use auditctl, on RHEL7, default is to use augenrules
+sed -i "s%^ExecStartPost=.*%ExecStartPost=-/sbin/auditctl%" /usr/lib/systemd/system/auditd.service
+
+rm -f /etc/audit/rules.d/*
+
+# cut out irrelevant rules for this test
+sed '1,10d' test_audit.rules > /etc/audit/audit.rules
+sed -i '5,8d' /etc/audit/audit.rules
+true
```
Comment (mildass) [inline tests/data/group_system/group_auditing/group_auditd_configure_rules/group_audit_kernel_module_loading/rule_audit_rules_kernel_module_loading/syscalls_multiple_per_arg.pass.sh]: I think that you forgot to remove `true` with last commit

Comment (yuumasato) [reply tests/data/group_system/group_auditing/group_auditd_configure_rules/group_audit_kernel_module_loading/rule_audit_rules_kernel_module_loading/syscalls_multiple_per_arg.pass.sh]: Fixed.

**Comment by mildass** [comment]
Looks good, thanks!

**Pull request merged**